closed/custom/copy/Copy-java.base.gmk

```diff
@@ -99,6 +98,11 @@ $(call openj9_copy_files,, \
 
 else ifeq ($(OPENJDK_TARGET_OS), zos)
 
+$(call openj9_copy_files_and_debuginfos, \
```

Comment:
Can you please adjust the comment on line 86: it's not just about 'static' libraries anymore.

Reply:
Updated